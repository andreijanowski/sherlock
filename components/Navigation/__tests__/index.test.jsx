/* eslint-env jest */
import React from "react";
import { shallow } from "enzyme";
import "jest-styled-components";
import { Nav } from "..";

const props = {
  t: m => m
};

describe("Nav component", () => {
  const comp = <Nav {...props} />;
  const shallowComponent = shallow(comp);

  it("renders correctly", () => {
    expect(shallowComponent).toMatchSnapshot();
  });
});
