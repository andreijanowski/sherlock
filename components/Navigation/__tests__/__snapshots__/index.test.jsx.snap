// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav component renders correctly 1`] = `
<Flex>
  navigation
</Flex>
`;
